lib/osf-components/addon/components/node-picker/dropdown-component/component.ts

```diff
@@ -0,0 +1,13 @@
+import Component from '@glimmer/component';
+
+/**
+ * The Dropdown Node Args
+ */
+interface DropdownNodeArgs {
+    isLoading: boolean;
+    hasMore: boolean;
+    loadMore: () => void;
+}
+
+// eslint-disable-next-line ember/no-empty-glimmer-component-classes
```

Comment:
Not sure if there's going to be stuff here in the future, but if not, I would be in favor of removing the empty glimmer component.ts file

Reply:
This component name is slightly misleading as well. This is the component for the "Load more" item, right? Not the dropdown itself?

Reply:
Let's leave this component file in here as a reference for valid component args.